contrib/hived-config-patcher/ds.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: hived-config-patcher-ds
+spec:
+  selector:
+    matchLabels:
+      app: hived-config-patcher
+  template:
+    metadata:
+      name: hived-config-patcher
+      labels:
+        app: hived-config-patcher
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: pai-master
+                    operator: In
+                    values:
+                      - "true"
+      hostNetwork: false
+      hostPID: false
+      containers:
+      - name: app
+        image: openpai/hived-config-patcher
+        imagePullPolicy: Always
+        command: ["python"]
+        args: [
+          "./patcher.py",
+          "--max-nodes={{cfg['openpai_worker_vmss']['max_vm_count']}}",
+          "--hived-config-file=/hived-config/hivedscheduler.yaml"
```

Comment:
Can the local file change be propagated to ConfigMap?

Reply:
no, added `patch_namespaced_config_map` to update configmap